compiler/rustc_typeck/src/errors.rs

```diff
@@ -270,7 +271,7 @@ impl<'a> SessionDiagnostic<'a> for MissingTypeParams {
 
         let mut suggested = false;
         if let (Ok(snippet), true) = (
```

Comment:
nit, can you use `let` chains?

```
let Ok(snippet) = ...
    && self.empty_generics
{
    ...
}
```

Reply:
Addressed here https://github.com/rust-lang/rust/pull/101434/commits/46ba27d5b54e4271e1c3934197afbae269237639.
Thanks!